src/main/java/games/strategy/engine/framework/startup/ui/editors/ForumPosterEditor.java

```diff
@@ -154,7 +154,7 @@ void testForum() {
       }
       poster.postTurnSummary(
           "Test summary from TripleA, engine version: " + ClientContext.engineVersion().getVersion()
-              + ", time: " + new SimpleDateFormat("HH:mm:ss").format(new Date()),
+              + ", time: " + TimeManager.getLocalizedTime(),
```

Comment:
:+1: Similar change as for `ChatMessagePanel#addChatMessage()`.  The message is simply being posted to a forum, so as long as no one parses those posts, we're ok.